interactors/search-field.js

```diff
@@ -0,0 +1,22 @@
+import { createInteractor, TextField, Select } from '@bigtest/interactor';
+
+const label = (el) => {
+  const labelText = el.querySelector('label');
+  return labelText ? labelText.innerText : undefined;
+};
+
+export default createInteractor('search field')({
+  selector: '[class^=searchField]',
+  locator: label,
+  filters: {
+    id: el => el.querySelector('input').getAttribute('id'),
+    readOnly: el => el.querySelector('input').hasAttribute('readOnly'),
+    value: el => el.querySelector('input').value,
+    placeholder: el => el.querySelector('input').placeholder,
+    isDisabled: el => el.querySelector('select').disabled,
```

Comment:
Convention is to not put `is` in front of filters, that way it reads nicely with the `is()` assertion: E.g.

```js
SearchField("users").is({ disabled: false })
```

```suggestion
    disabled: el => el.querySelector('select').disabled,
```